server/app/routers/api/router.js

```diff
@@ -7,10 +7,10 @@ const router = express.Router();
 /* ************************************************************************* */
 
 const usersRoutes = require("./users/router"); // Management des utilisateurs
-const tablesRoutes = require("./tables/router"); // Management des tableaux
```

Comment:
Why did you deleted tablesRoutes ? maybe that can affect a other feature of the project, you should not touch any code that in not in concern with your user story